stock_quant_package_restrict_disposable_reuse/models/stock_move_line.py

```diff
@@ -0,0 +1,80 @@
+# 2024 Copyright ForgeFlow, S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    result_package_id = fields.Many2one(
+        domain="['|', '|', ('location_id', '=', False), ('location_id', '=', location_dest_id),"
```

Comment:
We need to be conscient this is incompatible with another module doing the same.